python-sdk/src/astro/databases/google/bigquery.py

```diff
@@ -75,6 +75,15 @@ class BigqueryDatabase(BaseDatabase):
         FileLocation.S3: "load_s3_file_to_table",
         FileLocation.LOCAL: "load_local_file_to_table",
     }
+    AUTODETECT_SCHEMA_SUPPORTED: set[FileLocation] = {
+        FileLocation.GS,
+        FileLocation.LOCAL,
```

Comment:
@sunank200 can we change the Mapping to something like this - https://github.com/astronomer/astro-sdk/blob/ddf7f12b723595d88991d286fc39612cec6de455/python-sdk/src/astro/databases/google/bigquery.py#L62

Reply:
Isn't this dependent on filetypes as well?